docs/user-guide/meta-models.md

```diff
@@ -384,8 +384,21 @@ By putting these two together in an obvious way, we get the [`ZeroInflatedRegres
 ```
 
 ```console
-ZIR (RFC+RFR) r²: 0.8992404366385873
-RFR r²: 0.8516522752031502
+ZIR (RFC+RFR) r²: 0.8579468997736154
+RFR r²: 0.7691291933110612
+```
+
+If the underlying classifier is able to predict the _probability_ of a sample to be zero (i.e. it implements a `predict_proba` method), then the `ZeroInflatedRegressor` can be used to predict the probability of a sample being non-zero times the expected value of such sample.
+
+This quantity is sometimes called _risk estimate_ or _expected impact_, however, to adhere to scikit-learn convention, we made it accessible via the  `score_samples` method.
```

Comment:
cc: @edgBR 

Is this description reasonable to you? Any input is more than welcomed

Reply:
@edgBR reminder ping

Reply:
Hi guys, the description looks clear to me. However I will add that it can be called risk only if the classifier is calibrated.

Reply:
I will add a warning to mention that classifier should be calibrated